syntax = "proto2";

package rv.protos;

import "rastervision/protos/raster_source.proto";

message ComputeRasterStatsConfig {
    repeated RasterSource raster_sources = 1;
    optional string stats_uri = 2;
}
